CHANGES.md

```diff
@@ -9,6 +9,9 @@
 
 ## 10.0.4 (DR10, planned)
```

Comment:
Let's tag this as 10.1.0.

Reply:
Unless there are other significant DR10.1-related changes planned?

Reply:
Agreed. I've updated the `CHANGES.md` file accordingly.

Reply:
Note that there may be other DR10 changes (John's SGA changes spring to mind). But there shouldn't be any other specific changes for DR10.1.